www/src/hooks/use-community-events.js

```diff
@@ -0,0 +1,45 @@
+import { graphql, useStaticQuery } from "gatsby"
+
+const useCommunityEvents = () => {
+  const {
+    allAirtable: { nodes },
+  } = useStaticQuery(graphql`
+    {
+      allAirtable {
```

Comment:
given `Do_you_want_this_event_listed_on_the_gatsbyjs_org_events_page___public_page_` field in typedef - should this be used here to filter nodes?

Reply:
we could also filter by `Approved_for_posting_on_event_page` to avoid overfetching if we are filtering it out later on:
```
  const events = nodes
    .filter(event => event.data.approved)
```

Reply:
@pieh I just changed the typedefs to reflect what we're actually using. Good call on filtering, though — let me fix that.